BaseFilter/sections/antiadblock.txt

```diff
@@ -1401,16 +1401,22 @@ strtapeadblocker.art#@#.skyscraper.ad
 hotdebrid.com#%#//scriptlet('prevent-element-src-loading', 'script', 'ads.rubiconproject.com/ad/')
 !+ PLATFORM(ios, ext_android_cb)
 @@||ads.rubiconproject.com/ad/*.js$domain=hotdebrid.com
-! https://github.com/AdguardTeam/AdguardFilters/issues/123316
+! https://github.com/AdguardTeam/AdguardFilters/issues/133157
+work.ink#?#.content .col-md .row:has(div[id^="google_ads_"])
```

Comment:
Where do you need the rule?

Reply:
The rule had been tested when I wrote the exception rules.
However, the rule has been become useless due to the scriptlet rule.
I added a rule hiding leftovers on the website into `css_extended.txt`.